Server/Plugins/APIDump/Classes/Geometry.lua

```diff
@@ -1060,25 +1079,26 @@ the most popular tracing reasons - line of sight and solid hits.
 			{
 				Header = "Callbacks",
 				Contents = [[
-The Callbacks in the Trace() function is a table that contains named functions. Cuberite will call
+<p>The Callbacks in the Trace() function is a table that contains named functions. Cuberite will call
```

Comment:
The initial `<p>` is provided by the exporter.
```suggestion
The Callbacks in the Trace() function is a table that contains named functions. Cuberite will call
```